crates/aranya-crypto/src/afc/bidi.rs

```diff
@@ -218,6 +218,10 @@ pub struct BidiAuthorSecret<CS: CipherSuite> {
 
 sk_misc!(BidiAuthorSecret, BidiAuthorSecretId);
 
+impl<CS: CipherSuite> BidiAuthorSecret<CS> {
+    pub(crate) const CONTEXT: &'static str = "AFC Bidi Author Secret";
+}
```

Comment:
can you make CONTEXT part of `keypair!` (and therefore part of `secret_key!` and `kem_key!`)?